sources/packages/backend/workflow/test/2021-2022/parttime-assessment/eligibility/parttime-assessment-2021-2022-eligibility-BCAG.e2e-spec.ts

```diff
@@ -103,4 +104,59 @@ describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-eligibility-BCAG
       });
     }
   });
+
+  it("Should determine provincialAwardNetBCAGAmount when awardEligibilityBCAG is true", async () => {
+    // Arrange
+    const assessmentConsolidatedData =
+      createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+    assessmentConsolidatedData.studentDataCRAReportedIncome = 20001;
+    assessmentConsolidatedData.studentDataRelationshipStatus = "married";
+    assessmentConsolidatedData.studentDataIsYourSpouseACanadianCitizen =
+      YesNoOptions.Yes;
+    assessmentConsolidatedData.partner1CRAReportedIncome = 22999;
+    // Public institution
+    assessmentConsolidatedData.institutionType = InstitutionTypes.BCPublic;
+    // Act
+    const calculatedAssessment = await executePartTimeAssessmentForProgramYear(
+      PROGRAM_YEAR,
+      assessmentConsolidatedData,
+    );
+    // Assert
+    // calculatedDataTotalFamilyIncome = 20001 + 22999
+    // awardEligibilityBCAG is true
+    // federalAwardBCAGAmount is greater than or equal to 100
+    expect(calculatedAssessment.variables.calculatedDataTotalFamilyIncome).toBe(
+      43000,
+    );
+    expect(calculatedAssessment.variables.calculatedDataFamilySize).toBe(2);
+    expect(calculatedAssessment.variables.awardEligibilityBCAG).toBe(true);
+    expect(
+      calculatedAssessment.variables.federalAwardBCAGAmount,
+    ).toBeGreaterThan(100);
+    expect(calculatedAssessment.variables.provincialAwardNetBCAGAmount).toBe(
```

Comment:
can we please have the 1000 expected in this scenario to validate the  min(
     calculatedDataTotalRemainingNeed3,
     min(dmnPartTimeAwardAllowableLimits.limitAwardBCAGAmount, federalAwardBCAGAmount )
  )
  rather assigning it to 1000, directly?

Reply:
Sure. Good point! I've added this assertion in the latest commit. Please have a look.